decentraland/unity-explorer
# AudioSource SDKComponent

Add support for audio (sources) in SDK scenes.
- downloading, assigning and clean-up of the audio clips and respective audio sources
- budgeted and cached with supported cache clean-up
- add profiling markers for AudioClips
- unit, integration and one automated (big integration with real sdk scene) tests

# How to test
- start game and go to streaming-world
- switch to first-person to have proper audio spatialization
- verify you hear music
- run to dance floor
![image](https://github.com/decentraland/unity-explorer/assets/35366872/eda9a471-5713-41dd-b5e2-97bec7d5a698)
- verify you hear dance-floor music

### Note
- music not-playing outside of the scene is not implemented yet
- proper audio spatialization works stable in first person for now (because of how our cameras works)
